modules/ROOT/pages/events.adoc

```diff
@@ -140,7 +140,7 @@ The following events are provided by the {productname} editor.
 |PreProcess |`+{ node: Element, format: string }+` |Fired before serializing a DOM node to HTML content.
 |PostProcess |`+{ node: Element, format: string }+` |Fired after serializing a DOM node to HTML content.
 |SetAttrib |`+{ attrElm: Element, attrName: string, attrValue: string }+` |Fired when an attribute is updated using the editor xref:apis/tinymce.dom.domutils.adoc[DOMUtils API].
-|ResizeEditor |N/A |Fired when the editor is resized, either by the resize handles or the auto-resize plugin.
+|ResizeEditor |N/A |Fired when the editor is resized, either by the resize handles or the auto-resize plugin. In {productname} 6.1 and later, this event is also fired when the xref:fullscreen.adoc[`fullscreen`] plugin toggles fullscreen mode.
```

Comment:
```suggestion
|ResizeEditor |N/A |Fired when the editor is resized, either by the resize handles or the auto-resize plugin. In {productname} 6.1 and later, this event is also fired when fullscreen mode is toggled, with the xref:fullscreen.adoc[`fullscreen`] plugin enabled.
```
Small rewording, the fullscreen plugin only enables toggling fullscreen mode.

Reply:
I made a minor edit to your suggested re-write and pushed said minor edit. Thanks.